JSDemos/Demos/DataGrid/ToolbarCustomization/description.md

```diff
@@ -1 +1,13 @@
-The **DataGrid** widget allows you to&nbsp;customize its toolbar using the **onToolbarPreparing** function. You can add new items to&nbsp;the toolbar or&nbsp;customize the existing ones. In&nbsp;this demo, the toolbar contains a&nbsp;collection of&nbsp;other widgets from the DevExtreme HTML5&nbsp;UI Widgets library.
\ No newline at end of file
+The toolbar contains predefined and custom controls that execute various commands.
+
+The **DataGrid** uses the DevExtreme [Toolbar](/Demos/WidgetsGallery/Demo/Toolbar/Overview/) component as the toolbar. To customize the toolbar, implement the [onToolbarPreparing](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onToolbarPreparing) function. Its parameter contains the [Toolbar's configuration](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/) in the **toolbarOptions** field. You can access toolbar items via the **toolbarOptions**.[items](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/) array. Modify this array to add or remove toolbar items.
+
+This demo illustrates how to add the following elements to the toolbar:
+
+- **DevExtreme widgets**        
+Specify the [widget](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/#widget) that you want to add and its [options](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/#options). In this demo, we added the [Button](/Demos/WidgetsGallery/Demo/Button/PredefinedTypes/) and [SelectBox](/Demos/WidgetsGallery/Demo/SelectBox/Overview/) widgets.
```

Comment:
The way you write the first sentence looks like they need to specify widget's settings. "Specify the widget and its options". But I guess you specify the widget, together with its own settings. And you also specify the options for the toolbar item that contains the widget. 

In both sentences I'd use indefinite articles. 

```suggestion
Specify a [widget](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/#widget) that you want to display in a toolbar item and the item's [options](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/#options). In this demo, we extended the grid's command collection with a [Button](/Demos/WidgetsGallery/Demo/Button/PredefinedTypes/) and a [SelectBox](/Demos/WidgetsGallery/Demo/SelectBox/Overview/).
```

Reply:
A developer has to specify the widget name in the 'widget' option and this widget's options in the 'options' option. That's why the paragrpah is as follows:
```suggestion
Specify a [widget](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/#widget) that you want to add and its [options](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/#options). In this demo, we extended the toolbar's command collection with a [Button](/Demos/WidgetsGallery/Demo/Button/PredefinedTypes/) and a [SelectBox](/Demos/WidgetsGallery/Demo/SelectBox/Overview/).
```

Can it be applied?

Reply:
Yes, this version is good. I did some research - apparently I misunderstood what "options" were. Sorry.